src/content-helper/excerpt-generator/excerpt-generator.scss

```
@@ -1,47 +1,55 @@
 @import "../common/css/variables";
 @import "../common/css/functions";
 
-.editor-post-excerpt__spinner {
+.editor-post-excerpt__loading_animation {
 	position: absolute;
-	top: to_rem(20px);
-	left: 0;
-	right: 0;
-	bottom: 0;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	background: rgba(255, 255, 255, 0.7); // semi-transparent white background.
-	z-index: 10;
+	top: to_rem(29px);
+	left: to_rem(9px);
 }
 
-.editor-post-excerpt__spinner.has-word-count {
-	bottom: to_rem(20px);
+.editor-post-excerpt__textarea {
+	margin-bottom: to_rem(8px);
```

I would change this to `var(--grid-unit-10, to_rem(8px));`

```suggestion
	margin-bottom: var(--grid-unit-10, to_rem(8px));
```